modules/apps/object/object-test/src/testFunctional/tests/macros/ObjectAdmin.macro

```diff
@@ -10,10 +10,25 @@ definition {
 		Click(locator1 = "ObjectAdmin#SAVE_CUSTOM_OBJECT");
 	}
 
+	macro editCustomObject {
+		Click(
+			key_name = "${name}",
+			locator1 = "ObjectAdmin#SELECT_CUSTOM_OBJECT");
+
+		Click(locator1 = "ObjectAdmin#EDIT_CUSTOM_OBJECT");
+	}
+
```

Comment:
Please, remove it for now since we are doing it via API.

Reply:
@rodrigocunhaa  Done!